Tools/autotest/sim_vehicle.py

```diff
@@ -405,8 +405,12 @@ def do_build_waf(vehicledir, opts, frame_options):
 
     if opts.clean:
         run_cmd_blocking("Building clean", [waf_light, "clean"])
+        
+    if opts.jobs is None:
+        cmd_build = [waf_light, "build", "--target", frame_options["waf_target"] ]        
+    else:
+        cmd_build = [waf_light, "-j", str(opts.jobs), "build", "--target", frame_options["waf_target"] ]
```

Comment:
Minor: what about `cmd_build += ['-j', str(opts.jobs)]` if not None? That way we keep a single definition for the common options.